.github/SUPPORT.md

```diff
@@ -0,0 +1,4 @@
+# SAML-CTK support
+If you're looking for support for the SAML Conformance Test Kit, these are the best options:
```

Comment:
✏️ "These are the best options" sounds odd to me, especially with only one option. Maybe something like "..., go here:"?